src/unix/thread.c

```diff
@@ -130,13 +130,15 @@ closure_function(1, 0, void, run_thread,
 
 void thread_sleep_interruptible(void)
 {
+    disable_interrupts();
```

Comment:
Is this just here in preparation of having interrupts on in kernel? Interrupts should already be off, right?

Reply:
Yes, they will be off for now, though I think we should continue to write things as if interrupts could potentially be on for syscall / bh processing at some point.